src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -637,9 +642,14 @@ std::unique_ptr<data_iterator> activity_classifier::create_iterator(
 }
 
 std::unique_ptr<compute_context>
-activity_classifier::create_compute_context() const
+activity_classifier::create_compute_context(bool use_tensorflow) const
 {
-  return compute_context::create();
+  if (use_tensorflow) {
```

Comment:
This could use `read_state("use_tensorflow")` instead of adding an argument to the function, right?

Reply:
Yes